packages/itwin/grouping-mapping-widget/src/components/context/GroupHilitedElementsContext.ts

```diff
@@ -50,6 +58,8 @@ export const GroupHilitedElementsContext = React.createContext<GroupHilitedEleme
   setOverlappedElementsMetadata: () => {},
   setIsOverlappedColored: () => {},
   setIsVisualizationsEnabled: () => {},
+  setGroupColors: () => {},
+  groupColors: [],
```

Comment:
A `Map` would make this quicker and cleaner.
` backgroundColor={groupColors.find((c) => c.groupId === props.group.id)?.color ?? ""}`
vs
`backgroundColor={groupColors.get(props.group.id) ?? ""}`